test/test_deploy_templates.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import pytest
```

Comment:
If you split imports, there are usually three groups: standard library, third-party libs (installed dependencies) and local modules.

Reply:
Split to three groups.

Reply:
Yes, but:
* pathlib is in the standard library and should be in the same group as os and sys
* pytest is third-party and should be alone in a group.